src/components/Table/Table.test.tsx

```diff
@@ -376,4 +376,64 @@ describe("Render a Table component", () => {
     expect(rowValue0Text.textContent).toContain("John");
     expect(rowValue0Text.className).toBe("n-row-data n-table-no-divider");
   });
+  test("Table checkbox clickable but row is not clicked", async () => {
```

Comment:
Please update the test description to something more descriptive. e.g. "on clicking checkbox event should not be propagated to rowClick".